src/main/java/org/kcctl/service/KafkaConnectApi.java

```diff
@@ -18,12 +18,7 @@
 import java.util.List;
 import java.util.Map;
 
-import javax.ws.rs.DELETE;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
+import javax.ws.rs.*;
```

Comment:
As a best practice, let's avoid the star import, as it's prone to confusions with types from other packages.

Reply:
true, intellij did it, but I will take care